.nycrc.json

```diff
@@ -1,4 +1,12 @@
 {
   "extends": "@istanbuljs/nyc-config-typescript",
-  "reporter": ["lcovonly", "text-summary", "html"]
+  "reporter": [
+    "lcovonly",
+    "text-summary",
+    "html"
+  ],
+  "statements": 53,
+  "branches": 38,
+  "functions": 52,
+  "lines": 54
```

Comment:
These numbers are -1 the current values, trying to prevent regression for now, we can work on improving this as we go.

Reply:
Just curious on what was the issue here and why this was needed or what was the potential regression?

Reply:
We just didn't use the coverage in CI, specifying these numbers along with the `--check-coverage` flag in the npm script will make nyc exit with failure if we start testing less, so there's no regression here I want to start testing these since we're kicking off a number of changes for the major